### DISCUSSION TITLE
Payment Requests expires but keeps coming out to pay

### DISCUSSION THREAD

**Discussion by juniorcines** [btcpayserver/btcpayserver#6474] [Technical Support]
![image](https://github.com/user-attachments/assets/9e68d188-e4e2-48b1-80a0-7361a1c5869d)

Well, the Payment Requests links continue to appear when they expire to pay, it would be good to improve that part.

also the counter can be hidden or expired.

**Comment by pavlenex** [comment]
Are you still facing this issue, which version of BTCPay are you using?

**Comment by pavlenex** [comment]
Seems like an old issue, can't replicate, please re-open if you're still having issue with it.
